admin_guide/persistent_storage_nfs.adoc

```diff
@@ -74,30 +56,86 @@ persistent-volume.json
+released from their claim (i.e, after the user's `*PersistentVolumeClaim*` bound
+to the volume is deleted). Once recycled, the NFS volume can be bound to a new
+claim.
 
 == Automation
+As discussed, clusters can be provisioned with persistent storage using NFS in
+the following way:
+
+- Disk partitions can be used to link:#enforcing-disk-quotas[enforce storage
+quotas].
+- Security can be enforced by link:#volume-security[restricting volumes] to the
+namespace that has a claim to them.
+- link:#reclaiming-resources[Reclamation of discarded resources] can be
+configured for each persistent volume.
 
-In this topic, we covered how a cluster can be provisioned with persistent storage using NFS.  Disk partitions can be
-used to enforce storage quotas, security is enforced by restricting volumes to the namespace that has a claim to them, and
-reclamation of discarded resources can be configured for each persistent volume.
+They are many ways that you can use scripts to automate the above tasks. You can
+use an
+link:https://github.com/openshift/openshift-ansible/tree/master/roles/kube_nfs_volumes[example
+Ansible playbook] to help you get started.
+
+== SELinux and NFS Export Settings
+By default, SELinux does not allow writing from a pod to a remote NFS server.
+The NFS volume mounts correctly, but is read-only.
+
+To enable writing in SELinux on each node:
+
+----
+# setsebool -P virt_use_nfs 1 <1>
+----
+<1> `-P` makes the bool persistent between reboots.
 
-Many ways to script these tasks exist.  We've created an link:https://github.com/openshift/openshift-ansible/tree/master/roles/kube_nfs_volumes[Ansible playbook] to help you get started.
+Additionally, each exported volume on the NFS server itself should conform to
+the following:
 
+- Each export must be:
 
-== SELinux and NFS export settings
+----
+/example_fs *(rw,all_squash)
+----
 
-By default, SELinux will not allow writing from a pod to a remote NFS server.  The NFS volume would mount correctly but be read-only.  Enable writing in SELinux on each node with the following command:
+- Each export must be owned by *nfsnobody*:
 
-```
-# -P makes the bool persistent between reboots
-setsebool -P virt_use_nfs 1
-```
+----
+chmod nfsnobody:nfsnobody
+----
 
-Additionally, each exported volume on the NFS server itself should conform to the following:
+- Each export must have the following permissions:
 
-* each export must be: ```/example_fs *(rw,all_squash)```
-* each export is owned by nfsnobody: ```chmod nfsnobody:nfsnobody```
-* each export must have the permissions: ```chmod 700```
+----
+chmod 700
+----
 
-*Important!* The export above allows arbitrary users and clients to mount this volume.  Exports can be restricted to a range of IP addresses for hosts that will access the volume.
-   Refer to ```man exports``` for information.
+[IMPORTANT]
+====
+The export above allows arbitrary users and clients to mount this volume.
```

Comment:
On further reflection, probably shouldn't say "users and clients" but rather just "clients". While the lack of real security is a concern here (any pod could be configured to mount the volume directly if the user just knows what it is), the term "users" here is probably more confusing than helpful. Point is, the "*" leaves it open to any NFS client that can reach it from a network perspective, and most people will want to lock that down a bit.

Maybe:

    The export definition above allows arbitrary network clients to mount this volume.